### ISSUE TITLE
Invalid XML characters in JSON input

### ISSUE THREAD

**Issue by michaelhkay** [qt4cg/qtspecs#1667]
We have changed the data model (§2.8.4) (see PR #546) so that implementations may allow characters that are not valid XML characters.

We have not explored the impact of this change on parse-json(), which is one of the obvious places where non-XML characters may arise. For example, JSON allows unescaped C1 control characters.

(Note however that the data model explicitly bans unpaired surrogates, and I think that rule should apply to parse-json() even though the JSON grammar allows it).

Hopefully it only requires clarification notes to be added to the spec, and not any substantive change.

**Comment by ChristianGruen** [comment]
Good point. I have observed other cases in which a clarification would be helpful (I should spend some time to look at them closer).
